traveler-ios-ui/TravelerKitUI/ViewControllers/BookingOptionsViewController.swift

```diff
@@ -21,7 +21,7 @@ class BookingOptionsViewController: UIViewController {
 
     weak var delegate: BookingOptionsViewControllerDelegate?
 
-    var product: Product?
+    var product: BookingItemDetails?
```

Comment:
same here... Im actually not sure how we would go about this. But maybe theres an alternative here?

Reply:
The alternative I see is getting more elaborate with the protocols. But I feel this might make it difficult or might make some protocols a bit redundant.